sites/shared/prebuild/markdown.mjs

```diff
@@ -308,6 +308,28 @@ const writeFile = async (filename, exportname, site, content) => {
   )
 }
 
+/*
+ * Copy all the markdown resources + image assets to the public folder
+ */
+const copyMdRawFiles = async (site) => {
```

Comment:
We do not need to copy **all** markdown.
The only markdown that gets loaded dynamically are `docs`
So we can skip `blog`, `showcase`, and `newsletter` content.

As such, I'd like to update this to only copy the `docs` subfolder:
```mjs
  const src = await path.resolve(process.cwd(), '..', '..', 'markdown', site. 'org', 'docs')
  const dst = await path.resolve(process.cwd(), '..', site, 'public', 'markdown', 'org', 'docs')
```

Reply:
I made it copy everything to avoid the risk of someone changing something else in the future and running into the same problem again (e.g. if blog posts were changed to load dynamically), and because I wasn't sure exactly what parts were loaded from where (especially images which to me it looked like/seemed reasonable that they were always loaded remotely). (I'm not familiar with React so I have a hard time understanding what the code is actually doing.)

I don't agree with only copying docs, this sounds like a future issue waiting to happen, but if you want to change it then go ahead, I'm not sure why you need to ask me about it before merging.